docs/EXTENDED-ARCHIVES.md

```diff
@@ -162,78 +159,6 @@ You may declare the following colors, in hexadecimal format:
 
 The client decides how to use them and where, if at all.
 
-#### Device
```

Comment:
Is it a bit premature to be removing this? Isn't knowing what device(s) an image is suitable for actually quite valuable?